src/Types/PaymentType.res

```diff
@@ -130,6 +130,19 @@ type billingAddress = {
   usePrefilledValues: showType,
 }
 
+type sdkHandleConfirmPaymentProps = {
+  buttonText: string,
+  backgroundColor: string,
+  buttonHeight: string,
+  buttonWidth: string,
+  borderRadius: string,
+  borderColor: string,
+  textColor: string,
+  textFontSize: string,
+  textFontWeight: string,
+  confirmParams: ConfirmType.confirmParams,
+}
+
```

Comment:
keep the customization and other things separate

Reply:
Updated